pkg/satellite/validation.go

```diff
@@ -80,3 +80,23 @@ func validatePassword(pass string) error {
 
 	return errs.Combine()
 }
+
+// toLowerCase converts uppercase runes to lowercase equivalents
+// and returns resulting string
+func toLowerCase(s string) string {
```

Comment:
Rename to `normalizeEmail`. This is not for lower-casing the email.

Reply:
Unfortunately, email internationalization and unicode folding seems sufficiently bad as to warrant a clear description of the intent and possibly test cases.

The RFC 5321 says "The local-part of a mailbox MUST BE treated as case sensitive," but I believe assumes ASCII.  RFC 839 lowercases only ASCII.  Google removes periods and employs RFC 5233 subaddressing my.email+junk@gmail.com -- myemail@gmail.com

https://en.wikipedia.org/wiki/Email_address

Possibly see also "golang.org/x/text/cases"